prow/cmd/deck/static/common/rerun.ts

```diff
@@ -0,0 +1,151 @@
+  i.onclick = () => {
+    modal.style.display = "block";
+
+    rerunElement.innerHTML = `
+      <h2 class="rerunModal-title">Rerun ProwJob</h2>
+      <p class="rerunModal-description">
+        Below you can choose to rerun this ProwJob with either the original or latest configuration. 
+        <br><br> 
+        Note: Rerunning a ProwJob will create a new instance of a ProwJob. Any ProwJobs already running will not be interrupted.
+        <a href="${inrepoconfigURL}" target="_blank">
+          Inrepoconfig
+          <i class='rerunModal-openOut material-icons state triggered' style='color: gray'>open_in_new</i>
+        </a>
+        is currently not supported with the latest configuration option.
+      </p>
+      <div class="rerunModal-radioButtonGroup">
+        <div class="rerunModal-radioButtonRow">
+          <label class="rerunModal-radioLabel mdl-radio mdl-js-radio mdl-js-ripple-effect" for="rerunLatestOption">
+            <input type="radio" id="rerunLatestOption" class="mdl-radio__button" name="rerunOptions" checked>
+            <span class="mdl-radio__label">Latest Configuration</span>
+          </label>
+          (<a href="${latestURL}" target="_blank">View YAML
+          <i class='rerunModal-openOut material-icons state triggered' style='color: gray'>open_in_new</i>
+          </a>)
+        </div>
+        <div class="rerunModal-radioButtonRow">
+          <label class="rerunModal-radioLabel mdl-radio mdl-js-radio mdl-js-ripple-effect" for="rerunOriginalOption">
+            <input type="radio" id="rerunOriginalOption" class="mdl-radio__button" name="rerunOptions">
+            <span class="mdl-radio__label">Original Configuration</span>
+          </label>
+          (<a href="${originalURL}" target="_blank">View YAML
+          <i class='rerunModal-openOut material-icons state triggered' style='color: gray'>open_in_new</i>
+          </a>)
+        </div>
+      </div>
+      <div class="rerunModal-accordion">
+        <button class="rerunModal-accordionButton">
+          <i class='rerunModal-expandIcon material-icons state triggered' style='color: gray'>expand_more</i>
+          kubectl command
+        </button>
+        <div class="rerunModal-accordionPanel">
+          <div class="accordion-panel-content">
+            <p class="rerunModal-commandDescription">${getCommandDescription(LATEST)}</p>
+            <div class="rerunModal-commandContent">
+              <div class="rerunModal-command">${getCommand(commandURL)}</div>
+              <a class="rerunModal-copyButton mdl-button mdl-js-button mdl-button--icon">
+                <i class='material-icons state triggered' style='color: gray'>content_copy</i>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    `;
+
+    const rerunLatestOption = rerunElement.querySelector('#rerunLatestOption');
+    const rerunCommand = rerunElement.querySelector('.rerunModal-command');
```

Comment:
I feel like these could be named with `El` in their suffix.... but the usage and definition are so canonical, it's likely not necessary.

And since this is all local, I think the `rerun` prefix can be removed from "rerunXxx" local variables.

But maybe `accordion` => `accordionButton` and `accordionPanel` => `accordionEl`. (Now that I'm looking, I think I'd rename the input param `rerunElement` to `parentEl`. 😛 )

... Basically: feel free to make systematic changes to existing code, but do so safely and cleanly. 😁

Reply:
Updated to remove rerun prefix, and updated rerunElement to parentEl.